src/components/object-info/ObjectInfo.jsx

```diff
@@ -57,9 +69,10 @@ export function toExpandPathsNotation (localPath) {
 }
 
 const DagNodeIcon = ({ type, ...props }) => (
-  <svg {...props} title={nameForNode(type)} width='30' height='30' viewBox='0 0 30 30' xmlns='http://www.w3.org/2000/svg'>
-    <circle cx='15' cy='15' r='15' fillRule='evenodd' fill={colorForNode(type)} />
-  </svg>
+  null
+  // <svg {...props} title={nameForNode(type)} width='30' height='30' viewBox='0 0 30 30' xmlns='http://www.w3.org/2000/svg'>
+  //   <circle cx='15' cy='15' r='15' fillRule='evenodd' fill={colorForNode(type)} />
+  // </svg>
```

Comment:
oops. debugging amnesia. 

```suggestion
  <svg {...props} title={nameForNode(type)} width='30' height='30' viewBox='0 0 30 30' xmlns='http://www.w3.org/2000/svg'>
    <circle cx='15' cy='15' r='15' fillRule='evenodd' fill={colorForNode(type)} />
  </svg>
```